### PR TITLE
Added analytics configuration description.

### PR DESCRIPTION
**Pull request by Meruja**

### CODE REVIEW

File: en/docs/install-and-setup/setup/distributed-deployment/configure-apim-analytics/active-passive.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ then publishes them from where the active node left off. Once the terminated (pr
 operates in the passive state. In the passive node, sources are in an inactive mode where they will not receive events 
 into the system. 
 
+!!! note
+    The dashboard profile startup depends on the deployment. In Active-passive worker setup, You can have 2 JVMs for HA. Since you have only 2 nodes, it is fine to use the dashboard of the same binary (pack).
```
Comment (ruks) [inline en/docs/install-and-setup/setup/distributed-deployment/configure-apim-analytics/active-passive.md:19]: startup?? typo?

Comment (Meruja) [reply en/docs/install-and-setup/setup/distributed-deployment/configure-apim-analytics/active-passive.md]: Fixed with 9a8a2fb

**Review by ruks — APPROVED** [review]

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@Meruja , please change the font type of the image that you added to "Roboto" and send a PR with regard to the image.

**Comment by Mariangela** [comment]
@ruks , do we need to add these changes to 3.2.0 as well?

**Pull request merged**